README.md

```diff
@@ -1226,14 +1246,17 @@ But this window is not always open.
     [stand by the existing system](https://a16z.com/2016/06/23/options-timing/), arguing that it
     is better to incentivize people not to leave a company, or that long windows effectively
     transfer wealth from employees who commit long term to those who leave.
-  - As a side note, there is also an argument that it’s a legal requirement of ISOs to have a
-    90-day exercise window.
+  - As a side note, there is also an argument that it’s
+    [a legal requirement](https://news.ycombinator.com/item?id=9254299) of ISOs to have a 90-day
```

Comment:
This is great info from George but he doesn't make this argument; rather the link should be on the "true but misleading" phrase below to indicate what he says.

I'll do this.